app/views/posts/_create_comment.html.erb

```diff
@@ -0,0 +1,6 @@
+<%= form_with url: "/posts/#{@post.id}/comments", scope: :comment do |form| %>
+<div class="comment-container d-flex">
+	<%= form.text_area :text, placeholder: "Comment something here...", rows: 14, required: true %>
+	<%= form.submit "Comment", class: "submit-btn comment-btn" %>
+  </div>
+<% end %>
```

Comment:
- [ ] After creating a new action as discussed above. Please ensure that you generate views for comment. The form creating a comment should in `views/comments/new.html.erb`. It means that this form should not be under the `app/views/posts/` directory. We need to take care of separating concerns so that the app's maintainability can be affordable.

Reply:
@Lameck1 I sincerely apologize if this sounds bad. Acknowledging that I'm still a beginner, I don't see any sort of requirement to push the comment form away from posts view. It only asked us to make a comment form and in the sneak-peak they did not provide a template to follow regarding the matter so I felt free to do it in the way I kind of see in social medias and forums